cauldron/redis_cache.py

```diff
@@ -355,6 +371,20 @@ def set_key(self, key, value, namespace=None, expire=0):
                 key = self._get_key(namespace, key)
             yield from redis.set(key, value, expire=expire)
 
+    @coroutine
+    def set_key_sadd(cls, key, value, namespace=None):
```

Comment:
these should be self, let's remove these 2 methods from v2 for now. although I only told to put them here but we will add here as in when required